simulation_parameters/Constants.cs

```diff
@@ -46,6 +46,11 @@ public static class Constants
     // Should be the same as its counterpart in shaders/CompoundCloudPlane.shader
     public const float CLOUD_MAX_INTENSITY_SHOWN = 1000;
 
+    // length of cloud spawn cycle in seconds.
```

Comment:
Comment should start with a capital letter.

Reply:
Fixed.